src/app/ui/question/question.component.html

```diff
@@ -6,8 +6,36 @@ <h2>
   </h2>
 
   <div>
-    <button *ngIf="solvedQuestion$ | async" class="mt-2 btn btn-success btn-lg" (click)="unmarkAsSolved()" type="button"><fa name="check"></fa></button>
-    <button *ngIf="!(solvedQuestion$ | async)" class="mt-2 btn btn-outline-success btn-lg" (click)="markAsSolved();" type="button"><fa name="check"></fa></button>
+    <div *ngIf="course$ | async as course">
+      <div *ngIf="tags$ | async as tags">
+        <div *ngFor="let tag of tags">
+          <a class="tag" routerLink="/course/{{course.id}}/tag/{{tag}}">#{{tag}}</a>
```

Comment:
I suggest using https://getbootstrap.com/docs/4.3/components/badge/#pill-badges for the tags